web/src/components/vis/BoxplotPlot.vue

```diff
@@ -0,0 +1,130 @@
+<script>
+import { select } from 'd3-selection';
+import { scaleBand } from 'd3-scale';
+import { boxplot, boxplotStats } from 'd3-boxplot';
+import 'd3-transition';
+
+import { axisPlot } from './mixins/axisPlot';
+
+
+export default {
+  mixins: [
+    axisPlot,
+  ],
+  props: {
+    width: {
+      type: Number,
+      default: 400,
+    },
+    height: {
+      type: Number,
+      default: 300,
+    },
+    rows: { // {name: string, values: number[]}[]
+      type: Array,
+      default: () => [],
+    },
+  },
+  data() {
+    return {
+      margin: {
+        top: 20,
+        right: 20,
+        bottom: 50,
+        left: 120,
+      },
+      duration: 200,
+      ylabel: 'Metabolite',
+      xlabel: 'Measurement',
+      initialRun: false,
+    };
+  },
+  computed: {
+    reactiveUpdate() {
+      if (!this.initialRun) {
+        return '';
+      }
+      this.update();
+      return Math.random().toString();
```

Comment:
Is this necessary? AFAICT, computed properties are recomputed when their dependencies change, and I'm uncomfortable with making the value of this property change randomly, in case it were ever used as a dependency somewhere else.

@subdavis can you weigh in on this?